<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Slider</class>
 <widget class="QDialog" name="Slider">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Prefered unit</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="radio_value">
        <property name="text">
         <string>&amp;Value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_percent">
        <property name="text">
         <string>Pe&amp;rcent</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_ev">
        <property name="text">
         <string>Exposure Value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_mag">
        <property name="text">
         <string>Magnitude</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Scale</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QRadioButton" name="radio_linear">
        <property name="text">
         <string>&amp;Linear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_logarithmic">
        <property name="text">
         <string>Logarithmic</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Extrema and value</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QDoubleSpinBox" name="dspin_min">
          <property name="suffix">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="dspin_value">
          <property name="suffix">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="dspin_max"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="slider_value">
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Slider</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Slider</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_ev</sender>
   <signal>toggled(bool)</signal>
   <receiver>Slider</receiver>
   <slot>selectExposureValue(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_linear</sender>
   <signal>toggled(bool)</signal>
   <receiver>Slider</receiver>
   <slot>selectLinear(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_logarithmic</sender>
   <signal>toggled(bool)</signal>
   <receiver>Slider</receiver>
   <slot>selectLogarithmic(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_mag</sender>
   <signal>toggled(bool)</signal>
   <receiver>Slider</receiver>
   <slot>selectMagnitude(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_percent</sender>
   <signal>toggled(bool)</signal>
   <receiver>Slider</receiver>
   <slot>selectPercent(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_value</sender>
   <signal>toggled(bool)</signal>
   <receiver>Slider</receiver>
   <slot>selectValue(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dspin_min</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Slider</receiver>
   <slot>changeMin(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dspin_max</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Slider</receiver>
   <slot>changeMax(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dspin_value</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Slider</receiver>
   <slot>changeValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectValue(bool)</slot>
  <slot>selectPercent(bool)</slot>
  <slot>selectExposureValue(bool)</slot>
  <slot>selectMagnitude(bool)</slot>
  <slot>selectLinear(bool)</slot>
  <slot>selectLogarithmic(bool)</slot>
  <slot>changeMin(double)</slot>
  <slot>changeMax(double)</slot>
  <slot>changeValue(double)</slot>
 </slots>
</ui>
